build_scripts/tokenize

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python3
+"""
+This script parses a git diff report, fetches commit details, and tokenizes the results.
+
+This script takes three arguments:
+1. The path to `git_diff_report.txt`.
+2. The path to an intermediate file for storing diff information.
+3. The path to the output file for storing the tokenized results.
+
+Usage:
+    tokenize <git_diff_report_path> <intermediate_file_path> <output_file_path>
+"""
+
+import subprocess
+import re
+import json
+import argparse
+from datetime import datetime
+import tempfile
```

Comment:
Needs `import os` as well for the use of `os.remove`

Reply:
Fixed in PR2 - https://github.com/elisa-tech/delta-kernel/pull/3/files#diff-2a307bf1d08e3f4518d4887c595a9ff4eabd51094dab87f465790e1162fa3c17R18